test/client-errors.js

```diff
@@ -265,7 +265,7 @@ test('invalid URL throws', (t) => {
 })
 
 test('POST which fails should error response', (t) => {
-  t.plan(2)
+  t.plan(4)
```

Comment:
why this change?

Reply:
It didn’t call all the calllbacks before